src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -671,6 +672,10 @@ class MatterTestConfig:
 
     trace_to: List[str] = field(default_factory=list)
 
+    # Accepted Terms and Conditions if used
+    tc_version: int = None
+    tc_user_response: int = None
```

Comment:
Follow-up OK: suggest `tc_version_to_simulate` and `tc_user_response_to_simulate`.

Reply:
Updated